scripts/run-with-strimzi.sh

```diff
@@ -37,17 +54,54 @@ if [ $MINIKUBE_MEM_EXIT -eq 0 ]; then
 else
   echo "no minikube memory configuration, defaulting to 4096M"
 fi
-INSTALL_DIR="sample-install"
 ${MINIKUBE} start "${MINIKUBE_CONF}"
+
+NAMESPACE=kafka
+
+# Prepare kustomize overlay
+cp -r "${SAMPLE_DIR}" "${KUSTOMIZE_TMP}"
```

Comment:
As I want to edit the overlay (`${KUSTOMIZE} edit set image`), I found a needed to copy the kustomize tree, otherwise the developer would have a change in their local repo clone.   Anyone know a better way?

Reply:
copying off to some temporary dir to do the work sounds better to me, the local changes that it used to leave behind after runnig `sed` were asking for someone to accidentally commit the changes.

Reply:
It's the first time I've really used kustomize. It surprised me there was no way to do something like:
`kustomize build <dir> --override <blah>` to inject the image. 

The only way seems to be with an overlay.  You can't even put a temporary overlay in `/tmp` and point at a base/overlay in the tree as you hit directory traversal security constraints.

A copy recursive seemed like my only choice.